modules/transformer.js

```diff
@@ -52,6 +52,13 @@ export default class Transformer {
       const {
         type, variants, selectors, parse, insertMode = 'replace', params = {},
```

Comment:
`variants` are already supported with each block rule. Why do we need a separate list of variants?

Reply:
We can continue to discuss how the experience of actually applying variants during the mapping phase should look but I do not want to add extra complexity to the ruleset definition if it is not required.

Reply:
Also, if we are trageting block collection blocks first we should confirm if any of those blocks even support variants before we invest too much time in this area early on.